components/search/SearchForm.vue

```diff
@@ -34,15 +48,55 @@
         :aria-label="$t('search')"
         @input="getSearchSuggestions"
       />
+      <b-button
+        v-show="showSearch && showSearchQuery"
+        data-qa="clear button"
+        class="clear d-lg-none"
+        variant="light"
+        @click="clearQuery"
+      >
+        <span class="sr-only">
+          {{ $t('header.clearQuery') }}
+        </span>
```

Comment:
Could you see if it passes the accessibility test when this text is not in the span, but in an aria-label on the button? If that is fine, could you update the other buttons in this file too?

Reply:
moving to aria-labels does not throw accessibility errors :)